html/tooltip.html

```diff
@@ -91,20 +101,27 @@
         /* Trigger Tooltip with DOI information */
         {
             /* Set Scientific Name */
-            document.getElementById('tooltipScientificName').innerHTML =  record.data.attributes.referentName;
+            document.getElementById('tooltipScientificName').innerHTML =
+                `<a class="tooltipLink" target="_blank" href="https://dev.dissco.tech/ds/${doi}">${record.data.attributes.referentName}</a>`;
```

Comment:
These hardcoded URLs might become an issue later. However for the demo it should be fine

Reply:
Now I think about it, it will not change to sandbox if pushed to acceptance, so for demo on dev should be fine, otherwise not. Its probably not going to be in there forever.